source/intro.tex

```diff
@@ -333,34 +333,34 @@
 \indextext{conformance requirements!general|(}%
 The set of
 \defn{diagnosable rules}
-consists of all syntactic and semantic rules in this International
-Standard except for those rules containing an explicit notation that
+consists of all syntactic and semantic rules in this document
+except for those rules containing an explicit notation that
 ``no diagnostic is required'' or which are described as resulting in
 ``undefined behavior''.
 
 \pnum
 \indextext{conformance requirements!method of description}%
-Although this International Standard states only requirements on \Cpp
+Although this document states only requirements on \Cpp
```

Comment:
I must admit I find changing some of these places to "this document" a regression. (And I disagree with the ISO directives essentially banning "this International Standard".) I full agree with "this document" when it comes to internal organizations, chapters, etc. Once we get into defining requirements (such as above, or in "imposes no requirements on floating-point"), I feel "this Int. Std." makes it clearer that while it's the job of the Standard to define requirements, it opted not to for this particular case. (And other standards may fill the gap.) Just talking about "this document" seems too low-profile to me.
But ISO says we should use "this document", so my opinion is probably irrelevant.